codegen/src/main/resources/templates/rest-xml/s3/ExceptionUnmarshaller.ftl

```diff
@@ -0,0 +1,28 @@
+${fileHeader}
+package ${transformPackage};
+
+import org.w3c.dom.Node;
+import javax.annotation.Generated;
+
+import software.amazon.awssdk.core.AmazonServiceException;
+import software.amazon.awssdk.core.util.XpathUtils;
+
+import ${metadata.fullModelPackageName}.${shape.shapeName};
+
+@Generated("software.amazon.awssdk:aws-java-sdk-code-generator")
+public class ${shape.shapeName}Unmarshaller extends software.amazon.awssdk.services.s3.transform.S3ExceptionUnmarshaller {
```

Comment:
Import this?

Reply:
yeah, fixed!
> We'll need to refactor this when we refactor (un)marshallers but works for now.

yes, didn't know that. missed #30 :(